### PR TITLE
Fix Grievous rune for Crossbows

### PR DESCRIPTION
**Pull request by Drental**
also added another pass of localisation for the last PR that I initially missed :)

### CODE REVIEW

**Comment by stwlam** [comment]
hm

image

**Comment by Drental** [comment]
Make a third PR to split this? I noticed this after the other PR got merged

**Comment by Drental** [comment]
Oh! Now I see it

File: static/lang/re-en.json [hunk 1]
```diff
@@ -4117,6 +4118,7 @@
                     "bow": "Martial Bows",
                     "brawling": "Martial Brawling",
                     "club": "Martial Clubs",
+                    "crossbow": "Simple & Martial Crossbows",
```
Comment (Drental) [inline static/lang/re-en.json:4121]:
```suggestion
                    "crossbow": "Martial Crossbows",
```

**Pull request merged**